src/app/content/utils/archiveTreeUtils.ts

```diff
@@ -123,3 +123,18 @@ export const archiveTreeSectionIsChapter = (section: LinkedArchiveTreeNode): sec
   && !archiveTreeSectionIsBook(section)
   && getArchiveTreeSectionNumber(section) !== null
 ;
+
+/**
+ * If @param idA is before @param idB then return -1
+ * If @param idB is before @param idA then return 1
+ * In any other case return 0
+ */
+export const comparePositionsOfNodes = (tree: LinkedArchiveTree, idA: string, idB: string) => {
+  if (idA === idB) { return 0; }
```

Comment:
I feel like this can lead to some misleading results if the ids somehow happen to be in a different book. I don't have a strong opinion on it tho.

Reply:
I was thinking about this at I think it may be a good thing to not throw an error if ids are not found in the book, because we can't know if what we pass in the development is a valid child of a book so this can cause errors on the production (for example comparing ids returned from the api). I think returning 0 when ids were not found is okay, but we can wait for @TomWoodward to decide